### ISSUE TITLE
A11y_Model Builder_Image Classification_data_Keyboard: For Controls "Sample folders" and "Sub folders" present under Data preview "Tool tips" are not accessible with keyboard.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2656]
### GitHubTags
#WCAG2.1.1; #A11ySev3; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark; #Keyboard; 

### Environment Details
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps
1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu(Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Image Classification" and press enter.
9. Setup environemnt screen will appear, Tab till Local CPU tile and select it.
10. Tab till Next button and press enter.
11. Data screen will appear.
12. Tab till "Select a folder" and select the folder.
13. Now tab till "Data preview" section.
14. Now verify that For Controls "Sample folders" and "Sub folders" present under Data preview "Tool tips" are not accessible with keyboard.

### Actual Result
For Controls "Sample folders" and "Sub folders" present under Data preview "Tool tips" are not accessible with keyboard.
**Note: We are facing similar issue throughout the application.**

### Expected Result
For Controls "Sample folders" and "Sub folders" present under Data preview "Tool tips" tooltips should be accessible with keyboard.

### User Impact
Keyboard users will not be able to get the tooltip information if the tooltip is not accessible with keyboard.

### Attachment
![Bug_Tooltip](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/5910d21a-d8d4-4a28-a886-c64f0153cd92)

**Comment by zewditu** [comment]
@Harika1702  , ready to validate by Model Builder version 16.17.0.2332602

**Comment by zewditu** [comment]
@Harika1702 , ready to validate by Model Builder version 16.17.0.2332602

**Comment by Yogi2204** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
[Recording-20230629_170756.webm](https://github.com/dotnet/machinelearning-modelbuilder/assets/130456851/2afff6be-9b01-4b98-b4d2-65879584b976)
